src/nora/components/NoraTextAreaInput/NoraTextAreaInput.js

```diff
@@ -85,7 +84,7 @@ export const NoraTextAreaInput = ({
 
   // TODO: this indicates a field level error on the NoraTextAreaInput
   if (getError(currentError, touched)) {
-    console.log('NoraTextAreaInput field error: ', currentError)
+    // console.log('NoraTextAreaInput field error: ', currentError)
```

Comment:
I commented this out because I thought it looked weird to have any empty if statement, but don't feel strongly! I can also leave this in for now since we are not using `NoraTextAreaInput` at the field level yet.